src/parser.rs

```diff
@@ -47,17 +51,78 @@ where
     }
 }
 
-pub fn parse_meta_command(input: &str) -> Result<Option<MetaCommand>, ParserError> {
+pub fn parse_meta_command(input: &str) -> MetaCommand {
     if input.starts_with('.') {
+        if input.starts_with(".createdb") {
+            match parse_createdb(input) {
+                Ok(createdb_meta_command) => return createdb_meta_command,
+                Err(error) => return MetaCommand::MetacommandWithWrongArgs(MetaCommandError::ParseError(error.to_string())),
+            }
+        } else if input.starts_with(".dropdb") {
+            match parse_dropdb(input) {
+                Ok(dropdb_meta_command) => return dropdb_meta_command,
+                Err(error) => return MetaCommand::MetacommandWithWrongArgs(MetaCommandError::ParseError(error.to_string())),
+            }
+        }
+
         match input {
-            ".exit" | ".quit" => Ok(Some(MetaCommand::Exit)),
-            _ => Err(ParserError::UnknownMetaCommand(input)),
+            ".exit" | ".quit" => MetaCommand::Exit,
+            _ => MetaCommand::Unknown(input.to_string()),
         }
     } else {
-        Ok(None)
+        MetaCommand::Void
     }
 }
 
+pub fn parse_createdb(input: &str) -> Result<MetaCommand, ParserError> {
+    let current_dir = PathBuf::from("./");
```

Comment:
this probably should be config variable; probably can move into module const for now